CMakeLists.txt

```diff
@@ -213,42 +214,6 @@ if(CORE_SYSTEM_NAME STREQUAL osx)
   core_require_dep(Sdl)
 endif()
 
-if(NOT WIN32)
```

Comment:
as far as I can tell this block was only for X11 and Mir. this change looks ok to me